src/Footer.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+const StyledFooter = styled.footer`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 250px 0 290px;
+`;
+
+const StyledHref = styled.a`
+  text-decoration: none;
+  color: #636363;
+  font-size: 14px;
+`;
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: #636363;
+  font-size: 14px;
+`;
+
+const StyledIcon = styled.p``;
+
+function Footer() {
+  return (
+    <StyledFooter>
+      <StyledHref href="https://profy.dev/employers">profy.dev</StyledHref>
+      <StyledIcon>
+        <Link to="/">
+          <img src="./sign.svg" alt="" />
```

Comment:
I'd think you need to remove the `.` in the src prop. I mean use `/sign.svg`. Alternatively you can also import the svg as described [here])(https://create-react-app.dev/docs/adding-images-fonts-and-files/#adding-svgs)